packages/smooth_app/lib/pages/user_management/sign_up_page.dart

```diff
@@ -371,11 +371,12 @@ class _SignUpPageState extends State<SignUpPage> with TraceableClientMixin {
       return;
     }
     if (status.error != null) {
-      await LoadingDialog.error(context: context, title: status.error);
-
       // Highlight the field with the error
       if (status.statusErrors?.isNotEmpty == true) {
-        if (status.statusErrors!
+        if (status.statusErrors!.contains(SignUpStatusError.UNKNOWN)) {
+          //On Server Side Code this a new Error "SignUpStatusError.notValidEmail" needs to be added
```

Comment:
@Akashsri3bi Actually I see no connection between the issue #3226 and this code.
Here you just say "if it's an unknown error, put the focus on the email field" (which has no valid reason either).
How would that change the displayed error message?

Reply:
@monsieurtanuki I tried many cases and the error only appeared when we enter a wrong email **1st** reason to put focus on email field  , 
2 . SignupStatusError.Unknown is the only option for dealing with the error (for now ) as you recommended that we should not use signup.error any more .
3 . With this implementation , we won't need to show the error dialog but if we want it to be shown then I will go with dialog message only .

Reply:
@Akashsri3bi You should find inspiration in `SignUpStatus` (https://github.com/openfoodfacts/openfoodfacts-dart/blob/master/lib/model/SignUpStatus.dart) and add a fix there:
```dart
  /// Lists of errors sent by the server
  /// Sentences are available in
  /// [https://github.com/openfoodfacts/openfoodfacts-translations/blob/main/openfoodfacts-web/openfoodfacts-web-fr.po]
  static const Map<String, SignUpStatusError> _errorTexts =
      <String, SignUpStatusError>{
    'The e-mail address is already used by another user.':
        SignUpStatusError.EMAIL_ALREADY_USED,
    'This username already exists, please choose another.':
        SignUpStatusError.USERNAME_ALREADY_USED,
    'The password needs to be a least 6 characters long.':
        SignUpStatusError.INVALID_PASSWORD,
    'The password and confirmation password are different.':
        SignUpStatusError.INVALID_PASSWORD,
    'The user name must contain only unaccented letters, digits and dashes.':
        SignUpStatusError.INVALID_USERNAME,
  };
```

Reply:
okay , Thanks @monsieurtanuki !